packages/react-tei/src/SidePanel/keywords/Keywords.tsx

```diff
@@ -0,0 +1,14 @@
+import List from "@mui/material/List";
+import type { ComponentProps } from "../../tags/type.js";
+import { Value } from "../../tags/Value.js";
+
+export const Keywords = ({ data }: ComponentProps) => {
+	if (!data.value || data.value.length === 0 || !Array.isArray(data.value)) {
```

Comment:
Wrong order for params :p

```suggestion
	if (!data.value || !Array.isArray(data.value) || data.value.length === 0) {
```